codegen/end_to_end_test/lib/fragments/hero_with_interface_subtyped_fragments.graphql

```diff
@@ -0,0 +1,41 @@
+query HeroWithInterfaceSubTypedFragments($episode: Episode!) {
+  hero(episode: $episode) {
+    ...heroFieldsFragment
+  }
+}
+
+fragment heroFieldsFragment on Character {
+  id
+  name
+
+  ...on Human {
+    ...humanFieldsFragment
+  }
+
+  ...on Droid {
+    ...droidFieldsFragment
+  }
+}
+
+fragment humanFieldsFragment on Human {
+  homePlanet
+  birthday
+  friends {
+    ...on Droid {
+      id
+      name
+      ...droidFieldsFragment
+    }
+    #### NOTE: This cannot self-reference ...humanFieldsFragment hench the expanded fields
```

Comment:
Pretty sure we're not supporting recursive fragments... But wanted to test this out. ;-)